src/current/v23.2/admission-control.md

```diff
@@ -7,9 +7,23 @@ docs_area: develop
 
 CockroachDB supports an admission control system to maintain cluster performance and availability when some nodes experience high load. When admission control is enabled, CockroachDB sorts request and response operations into work queues by priority, giving preference to higher priority operations. Internal operations critical to node health, like [node liveness heartbeats]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#node-liveness-issues), are high priority. The admission control system also prioritizes transactions that hold [locks]({% link {{ page.version.version }}/crdb-internal.md %}#cluster_locks), to reduce [contention]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#transaction-contention) and release locks earlier.
 
-{{site.data.alerts.callout_info}}
-Admission control is not available for CockroachDB {{ site.data.products.serverless }} clusters.
-{{site.data.alerts.end}}
+## How admission control works
+
+At a high level, the admission control system works by queueing requests to use the following system resources:
+
+- CPU
+- Storage IO (writes to disk)
+
+For CPU, different types of usage are queued differently based on priority to allow important work to make progress even under high CPU utilization.
```

Comment:
Not sure if worth linking "high CPU" to: https://www.cockroachlabs.com/docs/stable/common-issues-to-monitor#cpu-usage

Up to you!

Reply:
Added the link! Always love to have more links!